{"name": "Prueba de login fallido", "status": "passed", "description": "Verifica que el login falle con credenciales incorrectas", "attachments": [{"name": "stdout", "source": "0a6815a3-0286-4973-8360-69d2bb90cb9c-attachment.txt", "type": "text/plain"}], "start": 1746894062755, "stop": 1746894063047, "uuid": "059cb47b-7564-47df-af23-80a2d676e571", "historyId": "f892cd1ad0909e3cab6ddca4d9ed5d9b", "testCaseId": "f892cd1ad0909e3cab6ddca4d9ed5d9b", "fullName": "EcoGuardian.tests.test_eco_guardian.AuthTests#test_login_fallido", "labels": [{"name": "parentSuite", "value": "EcoGuardian.tests"}, {"name": "suite", "value": "test_eco_guardian"}, {"name": "subSuite", "value": "AuthTests"}, {"name": "host", "value": "EdwinFigueroa10"}, {"name": "thread", "value": "21372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "EcoGuardian.tests.test_eco_guardian"}]}